src/DefaultHandle.jsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { withStyles, withStylesPropTypes } from 'react-with-styles';
+
+import { DEFAULT_HANDLE_WIDTH_UNITS, BACKGROUND_HEIGHT_UNITS } from './constants/SliderConstants';
+import handlePropTypes, { handleDefaultProps } from './propTypes/HandlePropTypes';
+
+export const propTypes = {
+  ...withStylesPropTypes,
+  ...handlePropTypes,
+  'aria-valuetext': PropTypes.string,
+  'aria-label': PropTypes.string,
+  getLabel: PropTypes.func,
+};
+
+const defaultProps = {
+  ...handleDefaultProps,
+  'aria-valuetext': undefined,
+  'aria-label': undefined,
+  getLabel: undefined,
+};
+
+function DefaultHandle({
+  css, styles, orientation, disabled, ...passProps
+}) {
+  const {
+    handleRef, theme, getLabel, ...rest
```

Comment:
same: if we break up the curly braces onto different lines, I think we should do the same for the contents